crypto/pem/pem_lib.c

```diff
@@ -93,6 +86,13 @@ int PEM_def_callback(char *buf, int num, int w, void *key)
         return (i);
```

Comment:
While you're touching this code anyway please replace all instances in this function of `return (...)` with `return ...` for compliance with the style guide.